home.html

```diff
@@ -0,0 +1,205 @@
+      integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous">
+    <link rel="stylesheet" href="CSS/style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato|Poppins|Anton">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+  </head>
+
+  <body>
+    <header>
+      <div class="socials container-fluid bg-dark text-light d-md-flex d-none justify-content-end px-3 gap-4 px-5">
+        <a href="#"><img src="Images/facebook.png" alt="Facebook"></a>
+        <a href="#"><img src="Images/twitter.png" alt="twitter"></a>
+        <span>English</span>
+      </div>
+      
+      <nav class="container-fluid navbar navbar-expand-md navbar-dark"> 
+        <div class="nav-bar container-fluid">   
+          <a class="navbar-brand logo text-center text-dark d-flex" href="#">
+            <img src="Images/logo.png" alt="Penda" class="logo-img py-1">
+            <span class="text-start ps-1 lato fw-bold">Ledership <br>Conference</span>
+          </a>     
+          <button class="navbar-toggler d-lg-none menu-btn" type="button" data-bs-toggle="collapse" data-bs-target="#collapsibleNavId" aria-controls="collapsibleNavId"
+            aria-expanded="false" aria-label="Toggle navigation">
+            <img src="Images/mobile_menu.png" alt="menu">
+          </button>
+          <div class="collapse navbar-collapse d-md-flex" id="collapsibleNavId">
+            <ul class="mobile-menu navbar-nav me-auto mt-2 mt-lg-0 bg-dark">
+              <li class="nav-item">
+                <a class="nav-link" href="about.html" aria-current="page">About</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Program</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Join</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Partners</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">News</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link quote red-border" href="#">PLC Campaign</a>
+              </li>             
+            </ul>      
+          </div>          
+        </div>
+      </nav>              
+    </header>
+    
+    <div class="pt-md-5">
+      <main class="container-1 section-container text-center text-md-start">
+        <div class="container-2 container d-flex row gap-1 p-5">
+          <p class="lato quote text-center container-fluid fs-5">"A genuine leader is not a searcher for consensus but a molder of consensus" ~ Martin Luther King, Jr.</p>
+          <h1 class="py-2 h1-effect"><span>PENDA LEADERSHIP CONFERENCE <br>2023</span></h1>
```

Comment:
- [ ] Please I would recommend you increase the `font size` of your `h1` by giving it something like `5em`. 👌